string_parser/string_parser.py

```diff
@@ -16,7 +16,27 @@ def count_words(self) -> dict:
         '''
         Takes a string and returns a dictionary with words quantity
         '''
-        words = self.str1.split()
+
+        list_of_punctuation_marks = [' ', ',', '.', '-']
+        words = []
+        punctuation_mark_count = 0
+
+        #Check if list contains punctuation marks. If so, split string at punctuation mark
+        for i in list_of_punctuation_marks:
+            if self.str1.find(i) != -1:
+                words = self.str1.split(i)
```

Comment:
str1 variable name tells me nothing about what it is. Please update all variables to be more descriptive